<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dWindowVerticalAxisSettingDialog</class>
 <widget class="QDialog" name="Graph2dWindowVerticalAxisSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vertical Axis Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="valueRangeGroupBox">
         <property name="title">
          <string>Value Range</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="autoRangeCheckBox">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="rangeMaxLabel">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="RealNumberEditWidget" name="rangeMaxLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="rangeMinLabel">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="RealNumberEditWidget" name="rangeMinLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="labelGroupBox">
       <property name="title">
        <string>Label</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="labelAutoRadioButton">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="labelCustomRadioButton">
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="labelCustomLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Graph2dWindowVerticalAxisSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Graph2dWindowVerticalAxisSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeMaxLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeMinLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelCustomRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCustomLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
